docs/documentation.md

```diff
@@ -33,22 +35,63 @@ $> rake db:migrate
 $> rails server
 ```
 
-Visit `http://localhost:3000/admin` and log in using:
+Visit `http://localhost:3000/admin` and voila: You&#8217;re on your brand
+new Active Admin dashboard.
 
-* __User__: admin@example.com
-* __Password__: password
+To register an already existing model, run:
 
-Voila! You&#8217;re on your brand new Active Admin dashboard.
+```bash
+$> rails generate active_admin:resource [MyModelName]
+```
 
-To register your first model, run:
+This creates a file at `app/admin/my_model_names.rb` for configuring the
+resource. Refresh your web browser to see the interface. In order to CRUD
+items, tweak the parameter `permit_params` in `app/admin/MyModel.rb`.
+
+## Usage with devise
+
+If you want to use the devise gem for (admin) authentification, add it to
+your Gemfile
+
+```ruby
+# Gemfile
+gem 'activeadmin'
+gem 'devise'
+```
+
+Update the bundle and run the installer
 
 ```bash
-$> rails generate active_admin:resource
-        [MyModelName]
+rails generate active_admin:install
 ```
 
-This creates a file at `app/admin/my_model_names.rb` for configuring the
-resource. Refresh your web browser to see the interface.
+Migrate your db
+
+```bash
+$> rake db:migrate
+```
+
+If you are adding devise with Active Admin, you need to seed the database
+with an admin user for Active Admin (otherwise you probably will already
+have an admin user)
```

Comment:
I would clarify here that the installer adds an admin user to the `seeds.rb` file, and that's it's up to the user whether to reseed the whole thing or manually create the admin user.